bgpd/bgpd.c

```diff
@@ -4223,8 +4223,6 @@ int bgp_delete(struct bgp *bgp)
 	if (bgp->process_queue)
 		work_queue_free_and_null(&bgp->process_queue);
 
-	event_master_free_unused(bm->master);
```

Comment:
Shouldn't this be merged to the same commit e94952fffb8354ff96914cd29591c9ceb7da66dc?

Reply:
ehhh they are two events.  a) remove the call in bgp to use a useless function.  b) remove the uselfess funcion